.tasks

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# When run automated, randomize to minimize stampeding herd
+if [ -t 0 ]; then
+    chance=10
+else
+    chance=$(shuf -i 0-10 -n 1)
+fi
+
+if [ $chance -gt 9 ]; then
+    ./tests/cleanup/remove_old_objects_aws.sh
+    ./tests/cleanup/remove_old_objects_azure.sh
+    ./tests/cleanup/remove_old_objects_openstack.sh
+    ./tests/cleanup/remove_old_objects_vmware.sh
```

Comment:
This will still run a magnitude of a dozen times a day. If that's too often, you could reduce the chance, and/or randomly pick one of the four cleanup tasks. But in general the structure looks fine.

Reply:
BTW, the "randomly pick one" can be achieved easily by prepending "echo" in front of all of these. Running ./.tasks is expected to produce a list of command lines to run, and the bot will randomly select one. Due to that you need to be careful -- if these programs output anything to stdout, that will be interpreted as commands to run.

So I'd advise to go with "echo" here to be on the safe side.